### ISSUE TITLE
Region issue

### ISSUE THREAD

**Issue by MAkerboom** [franciscogouveia/node-fritzbox#7]
`node-fritzbox online ip`

only works if language is set to German

I manually fixed it by editing the following line:
https://github.com/franciscogouveia/node-fritzbox/blob/master/modules/online.js#L87

OT: Thanks for your script.. it powers my Domoticz homeautomation by updating device status information. That way I know if any family members are at home.

**Comment by franciscogouveia** [comment]
Good catch. I'm in Germany, so the language on my Fritzbox is in German. When I find some time, I will try to find a way to do it language independent. For now, your quick fix should be enough.

Great to know someone found it useful! :) I'm using it on my server as well to update my DNS when my IP changes.

**Comment by FlorianWendelborn** [comment]
@franciscogouveia you can actually tell your FritzBox to send a HTTP GET request to your server, when your IP changes, that'd be most likely easier than what you're currently doing.

**Comment by franciscogouveia** [comment]
@dodekeract I'm updating the IP, because the server is behind the FritzBox and I want to access the server from outside (it's a home server).

What I could do is to configure FritzBox to update the DNS with the new IP every time it changes (e.g. with DynDNS). But, my domain provider does not support it.

The solution I found was delegating the DNS administration to AWS Route53. Like this, I created a script that checks if the IP changed, and if yes, send an update of the A record to Route 53 API.

**Comment by FlorianWendelborn** [comment]
@franciscogouveia ah, I didn't assume your server is behind the FritzBox, that makes sense then. I thought you're using a similar setup to mine, where I both have a home-server (rpi3) and a dedicated server in a datacenter.
